src/main/java/org/wso2/carbon/module/swiftiso20022/MT940FormatValidator.java

```diff
@@ -99,29 +99,29 @@ public void connect(MessageContext messageContext) {
      *
      * @param messageContext  Message context
      * @param lines           Payload lines
-     * @return                Error model
+     * @return                Validation Result
```

Comment:
Shall we improve these comment lines by explaining more?

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/8076e9b5033a9608b6839de38320ec994e01055d